project/plugins.sbt

```diff
@@ -4,3 +4,4 @@ addSbtPlugin("org.scalariform"    % "sbt-scalariform"       % "1.8.2")
 addSbtPlugin("org.scoverage"      % "sbt-coveralls"         % "1.2.7")
 addSbtPlugin("org.scoverage"      % "sbt-scoverage"         % "1.6.1")
 addSbtPlugin("org.xerial.sbt"     % "sbt-sonatype"          % "3.8.1")
+addSbtPlugin("org.scalameta"      % "sbt-mdoc"              % "2.2.9" )
```

Comment:
Any reason why we're not using the most recent 2.2.10 version?

Reply:
2.2.10 was released, I must have started doing this before that and didn't notice 😄 Updated on c1f0597fec5714d92afa83483925f676b94127e3.